### PR TITLE
Add additional USB device passthrough smartmontools options to webgui

### PR DESCRIPTION
**Pull request by jski**
There are some new devices since this page was last updated in the webgui, so I've gotten started with adding the ones that are available in version 7.3 (included with unraid 6.12, which I am using). I can't test all the flags as I don't have all of these devices available, but I can confirm that it does indeed work on my device using the flags added for the one device type I needed.

Hope this gets you started with getting this added into future versions!

### CODE REVIEW

File: emhttp/plugins/dynamix/DeviceInfo.page [hunk 1]
```diff
@@ -314,6 +314,13 @@ function setGlue(form,reset) {
    {glue:'' ,more:1,dev:1,type:'',min1:0,max1:15},                                         // hp cciss
    {glue:'' ,more:0,dev:0,type:''},                                                        // marvell
    {glue:',',more:1,dev:1,type:'',min1:0,max1:127}                                         // megaraid
```
Comment (ljm42) [inline emhttp/plugins/dynamix/DeviceInfo.page:316]: This needs a comma after `127}`

File: emhttp/plugins/dynamix/DeviceInfo.page [hunk 2]
```diff
@@ -314,6 +314,13 @@ function setGlue(form,reset) {
...
+   {glue:',',more:1,dev:0,type:'input'},min1:'',max1:'0xffffffff'},                        // cypress
```
Comment (ljm42) [inline emhttp/plugins/dynamix/DeviceInfo.page:317]: Looks like there is an extra `}` after `'input'`

**Review by jski — COMMENTED** [review]
Fixed

**Comment by limetech** [comment]
thank you!

**Pull request merged**